### ISSUE TITLE
openstack_ssh_public_key is not in the playbooks/provisioning/openstack/README.md file. 

### ISSUE THREAD

**Issue by mbruzek** [openshift/openshift-ansible-contrib#798]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

## **Problem Description**

The playbooks/provisioning/openstack/README.md file defines a variable `openstack_ssh_key` as a Nova keypair. There is no key named `openstack_ssh_key` in the provisioning/openstack/sample-inventory/group_vars/all.yml. The file all.yml does have a key named "openstack_ssh_public_key" and the README.md file should be updated with this string.

However this is very confusing because it appears "openstack_ssh_public_key" should be the Nova keypair name (not a path to an actual public key), and "openstack_private_ssh_key" is a path to a private key. 

## **Proposed Change**
Change the name of this variable to reflect more accurately what the variable should be set to, such as "openstack_keypair_name" or "openstack_keypair". 

### *Alternatives*
If making this change is too dangerous or risky we should comment the variable in all.yml and update the README.md file to reflect the correct variable name.

### *Documentation Impact*
Update playbooks/provisioning/openstack/README.md

## **Acceptance criteria**

The playbooks/provisioning/openstack/README.md file accurately describes  each openstack variable in all.yml so new developers can reference the README.md to correctly run the OpenStack provisioning step.

**Comment by tzumainn** [comment]
Interesting! I'd suggest something like openstack_ssh_public_key_name, as this value eventually gets used as the value for a Heat template parameter named key_name. I don't know if this will be the highest immediate priority, though.

**Comment by tomassedovic** [comment]
Good catch! As it happens, I've just created a pull request that updates the readme and fixes this issue: #799 

I'd like to spend some time cleaning up the variables and the sample inventories and this is a good suggestion. I'll try to make this happen before we do the first release.
